docs/extend/extend-cli/cli-installing-sample-plugin.md

```diff
@@ -3,13 +3,13 @@
 Before you begin, [set up](cli-setting-up.md) your local environment to install a plug-in.
 
 ## Overview
-This tutorial covers installing and running this bundled Zowe&trade; CLI plugin as-is (without modification), which will display your current directory contents. 
+This tutorial covers installing and running this bundled Zowe&trade; CLI plugin as-is (without modification), which displays your current directory contents.
 
-The plug-in adds a command to the CLI that lists the contents of a directory on your computer. 
+The plug-in adds a command to the CLI that lists the contents of a directory on your computer.
```

Comment:
Line 6 and 8 are kind of repetitive. It might be better to combine them to:
"This tutorial covers installing and running this bundled Zowe&trade; CLI plug-in as-is (without modification), which adds a command to the CLI that lists the contents of a directory on your computer."